### DISCUSSION TITLE
Routing to public internet

### DISCUSSION THREAD

**Discussion by uasui** [openthread/openthread#8308] [Q&A] [answered]
Hi,

I try to reach the internet by pinging a public ipv6 address from a end device running Openthread through Nordic Connect SDK.
This does not succeed, as the device does not know how to route it and drops it.

A `netdata show` results in something like

```
Prefixes:
[ML]::/64 paos low fffe
Routes:
[PD]://64 s med fffe
[PD]://62 s med fffe
Services:
...
```
with PD being a globally routable prefix from my ISP.

How is routing intended to be set up here? Is this an issue with IPv6, border router or end device?

Configuring a default route on a ot-br-posix example as suggested in https://groups.google.com/g/openthread-users/c/BhWQiCRPBAg/m/H1RucT3vBAAJ previously had worked. Can I do something similar on the end device, in case I don't control the border router?

**Comment by jwhui** [comment, marked as answer]
The border router needs to advertise a default route to reach an arbitrary IPv6 endpoint on the public Internet.

At the very least, the BR needs to advertise a route that has the same prefix as the IPv6 destination the device is trying to reach.

**Comment by uasui** [comment]
Thank you for clarifying these responsibilities.

And generally thank you for all the work you do, Jonathan. Your insights and knowledge sharing in particular is highly appreciated and inspiring! :1st_place_medal:
